### DISCUSSION TITLE
Anyone get GSEApy to work in AWS Sagemaker?

### DISCUSSION THREAD

**Discussion by xiaofwu** [zqfang/GSEApy#262] [Q&A]
I installed in a notebook at AWS Sagemaker. But it seems like the gseapy can not access to enrichr cloud from AWS. a simple excecution of gp.get_library_name() return error: ConnectTimeout: HTTPConnectionPool(host='maayanlab.cloud', port=80): Max retries exceeded with url: /Enrichr/datasetStatistics. Appreciate any feedback on the issue!

**Comment by zqfang** [comment]
Update the ENRICHR_URL will fix this.


```python
from gseapy.enrichr import Enrichr

er = Enrichr(..., organism="human", ....)
er.ENRICHR_URL = "https://maayanlab.cloud/"
## get full list of libraries
library_names = er.get_libraries()
## run the analysis
er.run()
```
